src/fundus/publishers/de/waz.py

```diff
@@ -44,3 +45,9 @@ def topics(self) -> List[str]:
             authors = generic_author_parsing(self.precomputed.meta.get("author"))
             topics = generic_topic_parsing(self.precomputed.meta.get("keywords"))
             return [topic for topic in topics if topic not in authors]
+
+    class V1_1(V1):
+        VALID_UNTIL = datetime.date.today()
+        _paragraph_selector = CSSSelector(".article-body > p")
```

Comment:
There is an author byline, that should be easy to remove since it has an attribure rel='author'. An example is here: https://www.waz.de/sport/fussball/article241952512/Rettig-weist-Kritik-am-DFB-aus-Politik-zurueck-Kenntnisfrei.html

Reply:
I added this.